From 6b555516dc97b0eb07b945d2cb98421d8a140dc8 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 6 Sep 2022 07:43:20 +0800
Subject: [PATCH 23/60] Revert "drm/vc4: dsi: Add correct stop condition to
 vc4_dsi_encoder_disable iteration"

This reverts commit cefc8e7e0ea4e11f750c4abbb6f5ebc8f85110d5.
---
 drivers/gpu/drm/vc4/vc4_dsi.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/vc4/vc4_dsi.c b/drivers/gpu/drm/vc4/vc4_dsi.c
index 0bda40c2d7879..43e86f0b55411 100644
--- a/drivers/gpu/drm/vc4/vc4_dsi.c
+++ b/drivers/gpu/drm/vc4/vc4_dsi.c
@@ -803,9 +803,6 @@ static void vc4_dsi_encoder_disable(struct drm_encoder *encoder)
 	list_for_each_entry_reverse(iter, &dsi->bridge_chain, chain_node) {
 		if (iter->funcs->disable)
 			iter->funcs->disable(iter);
-
-		if (iter == dsi->bridge)
-			break;
 	}
 
 	vc4_dsi_ulps(dsi, true);
-- 
2.25.1

